tokio-console/src/config.rs

```diff
@@ -152,6 +162,9 @@ pub struct ColorToggles {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
 struct ConfigFile {
+    target_addr: Option<String>,
+    env_filter: Option<String>,
```

Comment:
as per your question
> is it better a field name log instead of env_filter?

 i think we should probably call this
```suggestion
    log: Option<String>,
```

Reply:
I fixed: https://github.com/tokio-rs/console/pull/334/commits/9473b594cdae614261042c6cf533b79bfaa402da